ingress-nginx-controller.yaml

```diff
@@ -2,10 +2,50 @@
 package:
   name: ingress-nginx-controller
   version: 1.8.0
-  epoch: 5
+  epoch: 6
   description: "Ingress-NGINX Controller for Kubernetes"
   copyright:
     - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - ca-certificates
+      - curl
+      - curl-dev
+      - openssh-client
+      - openssl
+      - openssl-dev
+      - opentracing-cpp
+      - jaeger-cpp
+      - msgpack
+      - datadog-cpp
+      - yaml-cpp
+      - zipkin-cpp
+      - ssdeep
+      - luajit
+      - luajit-dev
+      - lua-cjson
+      - lua-resty-balancer
+      - lua-resty-cache
+      - lua-resty-cookie
+      - lua-resty-core
+      - lua-resty-dns
+      - lua-resty-global-throttle
+      - lua-resty-http
+      - lua-resty-ipmatcher
+      - lua-resty-lock
+      - lua-resty-memcached
+      - lua-resty-redis
+      - lua-resty-string
+      - lua-resty-upload
+      - brotli-dev
+      - gd-dev
+      - geoip-dev
+      - libmaxminddb-dev
+      - libxml2-dev
+      - libxslt-dev
```

Comment:
These -dev packages came in from one of the earlier commits but they don't look quite right to me.  I might try one more test and remove these or at least drop the `-dev` suffix